MARIE_AND_BERT/readme.md

```diff
@@ -0,0 +1,153 @@
+
+2. To recreate the files in `Dictionaries.zip`, please read [Dictionary creation](KGToolbox/readme.md) section `Dictionaries (DATA/Dictionaries folder)` to create the dictionary files. After the dictionary creation steps, the `DATA/Dictioanries` folder will be populated.
+
+3. To recreate the files in `EntityLinking.zip`, please first follow the steps in [EntityLinking Dataset creation](KGToolbox/EntityLinking/readme.md) to create training dataset for Entity Linking training and the follow the steps in [EntityLinking Training](KGToolbox/EntityLinking/readme.md) to train the Entity Linking model. After the steps, the `DATA/EntityLinking` folder will be populated.
+
+4. To recreate the files in `bert_pretrained.zip` 
+```python
+from pytorch_transformers.modeling_bert import BertModel
+model = BertModel.from_pretrained('bert-base-uncased')
+model.save_pretrained('/tmp/directory/for/models/')
+```
+The user will need to change `/tmp/directory/for/models` to their folder of choice and move the files to `DATA/bert_pretrained`
+
+5. By following step 2, the `label_dict.js` file will be created in `static/js/`
+
+  
+### Other Services
+To run the full functions of the Marie system, three other systems are required:
+
+1. The LDF server. See [LDF server readme](../JPS_LDF/README.md) to run it. 
+2. The semantic agents. See [PCE Agent readme](../Agents/PCEAgent/README.md) and [STDC Agent readme](../Agents/STDCThermoAgent/README.md) to create docker containers running them. 
+
+
+## Docker Deployment (Local)
+
+1. Make sure the `DATA` folder is populated following the steps in section `Required files` or `Recreation of required files`
+2. run `docker build  --no-cache -t marie_test .` to build the image
+3. run ` docker run -p 5003:80 -d marie_test:latest `. 
+4. The Marie web-interface will then be available at `http://localhost:5003` or `http://127.0.0.1:5003`
+
+
+## Docker Deployment (Server)
+For deployment on Linux server from scratch:
+
+1. Assume the user cloned the `TheWorldAvatar` git repository under `/home/user1/Marie`
+2. Load the files 2-6 mentioned in [Required files](#required-files) in `/home/user1/Marie/TheWorldAvatar/MARIE_AND_BERT/DATA`
+3. Created a folder `/home/user1/Marie/TheWorldAvatar/MARIE_AND_BERT/DATA/KG` . Create `ontospecies.nt` and `ontocompchem.nt` using 
+
+```
+python KGToolbox/SPARQLEndpoint/export_triples.py 
+--output_dir ontospecies 
+--endpoint http://www.theworldavatar.com/blazegraph/namespace/copy_ontospecies_marie 
+--output_filename ontospecies.nt
+```
+and
+```
+python KGToolbox/SPARQLEndpoint/export_triples.py 
+--output_dir ontocompchem 
+--endpoint http://www.theworldavatar.com/blazegraph/namespace/ontocompchem 
+--output_filename ontocompchem.nt
+```
+The script needs to be run under `/home/user1/Marie/TheWorldAvatar/MARIE_AND_BERT`. 
+4. Build a blazegraph image, see [Blazegraph container](https://github.com/lyrasis/docker-blazegraph#local-builds) for instructions. 
```

Comment:
For some reason, this enumeration doesn't render correctly: https://github.com/cambridge-cares/TheWorldAvatar/tree/dev-marie-and-bert/MARIE_AND_BERT